master/custom/templates/releasedashboard.html

```diff
@@ -46,4 +46,12 @@ <h4 class="panel-title">
             </div>
         </div>
     </div>
+    <div class="container">
+        <small>Generated at <time id="generatedAt" datetime="{{generated_at}}">{{generated_at}}</time></small>
```

Comment:
```suggestion
        <small>Generated at <time id="generatedAt" datetime="{{generated_at}}">{{generated_at}}</time> (cached for 5 minutes)</small>
```

Reply:
Maybe the cache duration should be a template parameter.

Reply:
No, this won't be cached for 5 minutes.